korangar/src/main.rs

```diff
@@ -87,6 +90,9 @@ fn main() {
     const DEFAULT_BACKGROUND_MUSIC: Option<&str> = Some("background_music\\01.mp3");
     const MAIN_MENU_CLICK_SOUND_EFFECT: &str = "¹öÆ°¼Ò¸®.wav";
 
+    // TODO: This should not be a constant.
```

Comment:
Can you maybe add to the comment what it should be instead or why it's fixed to 6?

Reply:
Thanks! Done :+1: